samples/javascript_nodejs/09.message-routing/bot.js

```diff
@@ -65,20 +65,20 @@ class MessageRoutingBot {
         // Create a dialog context
         const dc = await this.dialogs.createContext(context);
 
-        if(context.activity.type === ActivityTypes.Message) {
-            
+        if (context.activity.type === ActivityTypes.Message) {
+
             const utterance = context.activity.text.trim().toLowerCase();
 
             // handle conversation interrupts first
             const interrupted = await this.isTurnInterrupted(dc, utterance);
-            if(interrupted) {
-                return;
+            let dialogResult;
+            if (interrupted) {
+                dialogResult = interrupted;
```

Comment:
The return value from `this.isTurnInterrupted()` is a boolean, why is the dialogResult being set to this?